rosdep/base.yaml

```diff
@@ -2389,6 +2389,11 @@ libpqxx:
     trusty: [libpqxx-4.0]
     utopic: [libpqxx-4.0]
     vivid: [libpqxx-4.0]
+    wily: [libpqxx-4.0]
+    xenial: [libpqxx-4.0]
+    yakkety: [libpqxx-4.0v5]
+    zesty: [libpqxx-4.0v5]
+    artful: [libpqxx-4.0v5]
```

Comment:
The mandatory order is beneficial to ensure that separated edits don't collide as much as possible as well as. It also allows a human to scan through the hashes to search for a specific one in less than `O(n)`.